src/lib/temple/front/lock.ts

```diff
@@ -16,13 +14,9 @@ export async function getShouldBeLockedOnStartup() {
   }
 
   const closureTimestamp = Number(localStorage.getItem(CLOSURE_STORAGE_KEY));
-  const [shouldBackupMnemonic, autoLockTime] = await Promise.all([
-    fetchFromStorage<boolean>(SHOULD_BACKUP_MNEMONIC_STORAGE_KEY).catch(() => false),
-    getLockUpTimeout()
-  ]);
-  const shouldLockByTimeout = closureTimestamp && Date.now() - closureTimestamp >= autoLockTime;
+  const autoLockTime = await getLockUpTimeout();
 
-  return shouldLockByTimeout || shouldBackupMnemonic;
+  return closureTimestamp && Date.now() - closureTimestamp >= autoLockTime;
```

Comment:
Please check how backup of a new wallet works if a user closes the wallet by accident before a successful backup